### PR TITLE
Update hook-clr.py

### PR DESCRIPTION
**Pull request by denfromufa**
Find Python.Runtime.DLL in normal search paths and then in site-packages. Raise exception if not found!

### CODE REVIEW

File: PyInstaller/hooks/hook-clr.py [hunk 1]
```diff
@@ -16,11 +16,19 @@
...
-    library = ctypes.util.find_library('Python.Runtime')
-    # :todo: Should be issue a warning-message, if the libary is not
-    # found?
+    library = ctypes.util.find_library(pyruntime)
+    datas = []
     if library:
         datas = [(library, '')]
+    else:
+    	import site
+    	from os.path import join as pjoin, exists
```
Comment (htgoebel) [inline PyInstaller/hooks/hook-clr.py]: - Please but all imports at the top.
- Please do not use this unusual name `pjoin`, simply keep join.
- Please use `PyInstaller.compat.getsitepackages`

File: PyInstaller/hooks/hook-clr.py [hunk 2]
```diff
@@ -16,11 +16,19 @@
...
 if is_win:
-    library = ctypes.util.find_library('Python.Runtime')
-    # :todo: Should be issue a warning-message, if the libary is not
-    # found?
+    library = ctypes.util.find_library(pyruntime)
+    datas = []
     if library:
         datas = [(library, '')]
+    else:
+    	import site
```
Comment (htgoebel) [inline PyInstaller/hooks/hook-clr.py]: Please add a comment about what you are doing her and why.

**Comment by htgoebel** [comment]
Thanks for this pull-request. I added some line comments, though.

**Comment by denfromufa** [comment]
@htgoebel why is appveoyr build & testing failing on some python versions?

File: PyInstaller/hooks/hook-clr.py (hunk 2 above)
Comment (denfromufa) [inline PyInstaller/hooks/hook-clr.py]: done!

File: PyInstaller/hooks/hook-clr.py (hunk 1 above)
Comment (denfromufa) [inline PyInstaller/hooks/hook-clr.py]: done!

**Comment by htgoebel** [comment]
> why is appveoyr build & testing failing on some python versions?

I don't know, Appeoyr seams not the be quite stable, we have lot all errors with all the time.

**Comment by denfromufa** [comment]
@htgoebel have you tried wine builds and testing in addition to appveyor?

**Pull request merged**